hedera-mirror-importer/src/main/resources/db/migration/v1/V1.39.0__support_nfts.sql

```diff
@@ -0,0 +1,44 @@
+-------------------
+-- Update and add tables for NFT support
+-------------------
+
+-- Update the token table
+alter table token
+    add column max_supply bigint not null default 9223372036854775807, -- max long
+    add column supply_type character varying(10) not null default 'INFINITE',
+    add column type character varying(20) not null default 'NON_FUNGIBLE_UNIQUE';
+
+-- Create nft table
+create table if not exists nft
+(
+  account_id            bigint                  not null,
+  created_timestamp     bigint  primary key     not null,
+  deleted               boolean default false   not null,
+  modified_timestamp    bigint                  not null,
+  metadata              bytea   default ''      not null,
+  serial_number         bigint                  not null,
+  token_id              bigint                  not null
+);
+create unique index if not exists nft__token_id_serial_num
+    on nft (token_id, serial_number);
+comment on table nft is 'NFT';
```

Comment:
nit: Something more suggestive. Template suggestions below that you can modify.
```suggestion
comment on table nft is 'Non Fungible Tokens (NFTs) minted on network';
```

Reply:
I like that, just made `Non Fungible` into `Non-Fungible`